pd-client/client.go

```diff
@@ -556,6 +556,9 @@ type TSFuture interface {
 }
 
 func (req *tsoRequest) Wait() (int64, int64, error) {
+	// If tso command duration is observed very high, the reason could be it
+	// takes too long for Wait() be called.
+	cmdDuration.WithLabelValues("tso_wait").Observe(time.Since(req.start).Seconds())
```

Comment:
Will it take a long time before call `Wait()`?

Reply:
Yes. In TiDB, some other tasks are performed while GetTS is running, such as plan building. If other tasks execute slow, it may take some time before calling `Wait()`.